packages/fxa-auth-server/lib/sentry.js

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Hoek = require('@hapi/hoek');
+const Sentry = require('@sentry/node');
+
+const getVersion = require('./version').getVersion;
+
+// Matches uid, session, oauth and other common tokens which we would
+// prefer not to include in Sentry reports.
+const TOKENREGEX = /[a-fA-F0-9]{32,}/gi;
+const FILTERED = '[Filtered]';
+const URIENCODEDFILTERED = encodeURIComponent(FILTERED);
+
+/**
+ * Filters all of an objects string properties to remove tokens.
+ *
+ * @param {Object} obj Object to filter values on
+ */
+function filterObject(obj) {
+  if (typeof obj === 'object') {
+    for (const [key, value] of Object.entries(obj)) {
+      if (typeof value === 'string') {
+        obj[key] = value.replace(TOKENREGEX, FILTERED);
+      }
+    }
+  }
+  return obj;
+}
+
+/**
+ * Filter a sentry event for PII in addition to the default filters.
+ *
+ * Current replacements:
+ *   - A 32-char hex string that typically is a FxA user-id.
+ *
+ * Data Removed:
+ *   - Request body.
+ *
+ * @param {Sentry.Event} event
+ */
+function filterSentryEvent(event, hint) {
+  if (event.breadcrumbs) {
+    for (const bc of event.breadcrumbs) {
+      bc.message = bc.message.replace(TOKENREGEX, FILTERED);
```

Comment:
Locally, `message` is undefined for me, https://sentry.prod.mozaws.net/operations/fxa-ben-test/issues/6540571/?query=is:unresolved